docs/resources/interface.md.erb

```diff
@@ -30,61 +31,71 @@ An `interface` resource block declares network interface properties to be tested
       it { should be_up }
       its('speed') { should eq 1000 }
       its('name') { should eq eth0 }
+      its('ipv4_addresses') { should include '10.0.0.5' }
     end
 
 <br>
 
 ## Properties
 
-`name`, `speed`
+`ipv4_address`, `ipv4_addresses`, `ipv4_addresses_netmask`, `ipv4_cidrs`, `ipv6_addresses`, `ipv6_cidrs`, `name`, `speed`
 
-<br>
 
 ## Resource Property Examples
 
-### name
-
-The `name` property tests if the named network interface exists:
+### ipv4_address
 
-    its('name') { should eq eth0 }
+Provided for compatibility with ServerSpec. Returns the first `ipv4_cidrs` entry.
 
-### speed
-
-The `speed` property tests the speed of the network interface, in MB/sec:
-
-    its('speed') { should eq 1000 }
+    its('ipv4_address') { should eq '10.0.0.5/8' }
```

Comment:
How come the IP addresses appear in CIDR form? Is the mask number `8` just redundant here?

Reply:
The `ipv4_address` property is provided for compatibility with ServerSpec, and that is how ServerSpec returns the value. Compatibility was requested on #722.

https://serverspec.org/resource_types.html#interface

Reply:
#722 seems concerned with getting the IP address, and gives the serverspec form as an example. From reading through the issue, I think the `8` was just a detail in the serverspec example but it's not beneficial for us to include. 

I imagine that people writing controls for IP addresses will want to work with just the IP address and needing to also include the mask number in their expected IP address string is just extra work for them.

We're also implementing `ipv4_addresses_netmask` and that lets users test IP address and mask if they need to additionally test that the mask is correct.

Reply:
As discussed, please have this property only return the IP address, and drop the mask number.

I said I would raise an issue to change our documentation about serverspec compatibility - I see there's already https://github.com/inspec/inspec/issues/4371 for this